pricelist_per_product/pricelist.py

```diff
@@ -91,12 +93,39 @@ def button_product(self):
         }
 
     @api.model
+    def update_sequence(self, vals):
```

Comment:
1/ You should explain why this method with a docstring
2/ method name could be in collision with some method in other module: what do you think of `update_sequence_according_to_product_or_product_tmpl` or something like that